hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/parse/QueryConverter.java

```diff
@@ -292,4 +272,40 @@ public Void visitSubQuery(RexSubQuery subQuery) {
 
         return new NestedExistsFinder().find();
     }
+
+    // Note: it must be used only in static class initializer.
```

Comment:
this note is too strict, this method could be used anywhere

Reply:
I'd move the whole code to the static initializer, but it doesn't look nice, so I moved it to the method.
It's **highly** **unlikely** to **reuse** this method **anywhere**.

Reply:
but then you could say "it is *intended to be used* only in static class initializer."